R/packages.R

```diff
@@ -23,3 +23,43 @@
     )
   }
 }
+
+# Prevent R CMD check from complaining about the use of pipe expressions
```

Comment:
FYI, there is a dedicated section `Dealing with “undefined global functions or variables”` in the vignette (https://cloud.r-project.org/web/packages/data.table/vignettes/datatable-importing.html).

Reply:
We can consider changing it, @j-smola suggested something similar. But for now I would prefer to use solution which we have in all packages

Reply:
Please add an evaluation task then (in which we will try to work out the best solution across all our packages).

Reply:
added [GDR-1947](https://jira.gene.com/jira/browse/GDR-1947)